### PR TITLE
St 94/feature/task display on myday page ui

### PR DESCRIPTION
**Pull request by jayDesai89**

### CODE REVIEW

**Comment by jayDesai89** [comment]
1. After Git pull, Run NPM Install
2. What to expect UI wise
→ Lend on myday page, have a list of all task, Change a view from grid ↔ list button
→ Lend on Planner day, same  UI as explained above
3. What to expect Code wise
→ Independent component for Taks and to-do display
→  Communication between parent ↔ child component for data sharing

File: src/app/components/task-display/task-display.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+
+import { TaskDisplayComponent } from './task-display.component';
+
+describe('TaskDisplayComponent', () => {
```
Comment (Dhruvkumarshah) [inline src/app/components/task-display/task-display.component.spec.ts:6]: Remove all the spec files

File: src/app/myday/myday.page.ts [hunk 2]
```diff
@@ -12,6 +12,43 @@ import { AlertReminderService } from './services/alert-reminder.service';
   styleUrls: ['./myday.page.scss']
 })
 export class MydayPage implements OnInit, OnDestroy {
+ /**
+  * Below allTask array is a mock data for UI purpose only
+  * 
+  */
+  allTask = [{
+    id: 1,
+    category: 'task',
```
Comment (Dhruvkumarshah) [inline src/app/myday/myday.page.ts:21]: Why all task have this json structure? This is not the correct one. Please update with existing object. As well all services are working for my day so we do not need mock data.

**Review by raxitp02 — COMMENTED** [review]
Resolve Conflict first. @jayDesai89

**Review by raxitp02 — APPROVED** [review]

File: src/app/components/task-display/task-display.component.spec.ts (hunk 1 above)
Comment (jayDesai89) [reply src/app/components/task-display/task-display.component.spec.ts:6]: why?

File: src/app/myday/myday.page.ts (hunk 2 above)
Comment (jayDesai89) [reply src/app/myday/myday.page.ts:21]: The scope of this ticket does not cover establishing contract between service and component. Mock data is solely present there to display the UI, with another functionality task, I will add the contract.

**Review by Sharmil7171 — APPROVED** [review]

**Pull request merged**